lib/deepstream/record.rb

```diff
@@ -14,6 +12,10 @@ def initialize(client, name)
       @client.send_message(TOPIC::RECORD, ACTION::CREATEORREAD, @name)
     end
 
+    def get_name
```

Comment:
You could have used record.data['name'] to get a value when a record's data contained a key 'name'. Right now you can't get the whole data hash at all (whether it should by always a hash, however, is a different issue).